packages/wallets/src/components/SentEmailContent/SentEmailContent.tsx

```diff
@@ -56,6 +56,16 @@ const SentEmailContent: React.FC<TProps> = ({ description, platform }) => {
         if (count === 0) setHasCountdownStarted(false);
     }, [count]);
 
+    const { data: activeWallet } = useActiveWalletAccount();
+
+    let redirectTo = platform === 'mt5' ? 1 : 2;
+
+    if (activeWallet?.is_virtual) {
+        redirectTo = Number(`${redirectTo}1`);
+    } else {
+        redirectTo = Number(`${redirectTo}0`);
+    }
```

Comment:
There are some similar blocks of code below, can you merge them together in a util function?